ios/RNSentryRNSScreen.m

```diff
@@ -0,0 +1,26 @@
+#import <Sentry/SentryFramesTracker.h>
+#import <Sentry/SentryDependencyContainer.h>
+#import <Sentry/SentrySwizzle.h>
+
+#import "RNSentryRNSScreen.h"
+#import "RNSentryDependencyContainer.h"
+
+@implementation RNSentryRNSScreen
+
++ (void)swizzleViewDidAppear {
```

Comment:
`h`: Using swizzling to start the frames tracker seems weird to me. Why did you choose that approach?

Reply:
The frame tracker is already running ([initialized here](https://github.com/getsentry/sentry-react-native/blob/dfadf15d2fb88910293eec14f78bf8c9670b90f6/ios/RNSentryDependencyContainer.m#L30)), 
but the listener is added in the swizzled `viewDidAppear` of the RNSScreen ([added listener](https://github.com/getsentry/sentry-react-native/blob/dfadf15d2fb88910293eec14f78bf8c9670b90f6/ios/RNSentryFramesTrackerListener.m#L27)).

Reply:
Why do you need the listener? We can also discuss this on a call and document the outcome here. I'm unsure if it's required.